net/http/inc/THttpServer.h

```diff
@@ -44,7 +44,8 @@ class THttpServer : public TNamed {
    TString fDefaultPageCont; ///<! content of the file content
    TString fDrawPage;        ///<! file name for drawing of single element
    TString fDrawPageCont;    ///<! content of draw page
-   TString fCors;            ///<! CORS (cross-origin resource sharing): sets Access-Control-Allow-Origin for ProcessRequest responses
+   TString
```

Comment:
This one looks weird to me...

Reply:
It is result of clang-format. 
Probably, due to long comments